config/config.json

```diff
@@ -93,7 +103,9 @@
         "bugprone-argument-comment" : ["sensitive", "extreme"],
         "bugprone-assert-side-effect" : ["default", "sensitive", "extreme"],
         "bugprone-bool-pointer-implicit-conversion" : ["default", "sensitive", "extreme"],
+        "bugprone-copy-constructor-init" : ["default", "sensitive", "extreme"],
         "bugprone-dangling-handle" : ["default", "sensitive", "extreme"],
+        "bugprone-exception-escape" : ["default", "sensitive", "extreme"],
```

Comment:
Are we sure we want to add this to the default set?
There is a warning for this checker `WARNING! This check may be expensive on large source files.`
https://clang.llvm.org/extra/clang-tidy/checks/bugprone-exception-escape.html

Reply:
put to sensitive, extreme only. We will need to measure this to put into the default profile.